core/iwasm/libraries/libc-wasi/sandboxed-system-primitives/src/posix.c

```diff
@@ -323,37 +325,47 @@ __wasi_errno_t
 wasmtime_ssp_clock_res_get(__wasi_clockid_t clock_id,
                            __wasi_timestamp_t *resolution)
 {
-#ifdef BH_PLATFORM_WINDOWS
-    return __WASI_ENOSYS;
+
+    #ifdef BH_PLATFORM_WINDOWS
+    //os_win_clock_res_get(clock_id, resolution);
+
+
+
 #else
-    clockid_t nclock_id;
-    if (!convert_clockid(clock_id, &nclock_id))
-        return __WASI_EINVAL;
-    struct timespec ts;
-    if (clock_getres(nclock_id, &ts) < 0)
-        return convert_errno(errno);
-    *resolution = convert_timespec(&ts);
+    {
+         __wasi_clockid_t nclock_id;
+        if (!convert_clockid(clock_id, &nclock_id))
+            return __WASI_EINVAL;
+        struct timespec ts;
+        if (clock_getres(nclock_id, &ts) < 0)
+            return convert_errno(errno);
+        *resolution = convert_timespec(&ts);
+    }
+
     return 0;
 #endif
+    return true;
 }
 
 __wasi_errno_t
-wasmtime_ssp_clock_time_get(__wasi_clockid_t clock_id,
-                            __wasi_timestamp_t precision,
+wasmtime_ssp_clock_time_get(__wasi_clockid_t clock_id, __wasi_timestamp_t precision, 
                             __wasi_timestamp_t *time)
-{
-#ifdef BH_PLATFORM_WINDOWS
-    return __WASI_ENOSYS;
-#else
-    clockid_t nclock_id;
-    if (!convert_clockid(clock_id, &nclock_id))
-        return __WASI_EINVAL;
-    struct timespec ts;
-    if (clock_gettime(nclock_id, &ts) < 0)
-        return convert_errno(errno);
-    *time = convert_timespec(&ts);
+{ 
+    #ifdef BH_PLATFORM_WINDOWS 
```

Comment:
similarly to `wasmtime_ssp_clock_res_get`, we could have `os_clock_time_get` in a platform-specific file. Please see my comment above.